.github/workflows/Tests.yaml

```diff
@@ -515,6 +515,8 @@ jobs:
         # container which arises from using the "--privileged" flag just below.
         OMPI_ALLOW_RUN_AS_ROOT: 1
         OMPI_ALLOW_RUN_AS_ROOT_CONFIRM: 1
+        # The number of cores to run on
```

Comment:
Mind adding the link to https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories